migrate/troubleshooting.md

```diff
@@ -219,4 +220,54 @@ Timescale instance or "merge" source databases to target schemas.
 
 The tsdbadmin database user is the most powerful available on Timescale, but it
 is not a true superuser. Review your application for use of superuser privileged
-operations and mitigate before migrating.
\ No newline at end of file
+operations and mitigate before migrating.
+
+## Debugging background jobs
+
+By default, background workers do not print a lot of information about
+execution. The reason for this is to avoid writing a lot of debug
+information to the PostgreSQL log unless necessary.
+
+To aid in debugging the background jobs, it is possible to increase
+the log level of the background workers without having to restart the
+server by setting the `timescaledb.bgw_log_level` GUC and reloading
+the configuration.
+
+```sql
+ALTER SYSTEM SET timescaledb.bgw_log_level TO 'DEBUG1';
+SELECT pg_reload_conf();
+```
+
+This variable is set to the value of [`log_min_messages`][1] by
+default, which typically is `WARNING`. If the value of
+[`log_min_messages`][1] is changed in the configuration file, this
+value will be used instead for `timescaledb.bgw_log_level`.
+
+[1]: https://www.postgresql.org/docs/current/runtime-config-logging.html#GUC-LOG-MIN-MESSAGES
+
+### Debug level 1
+
+The amount of information printed at each level varies between jobs,
+but the information printed at `DEBUG1` is currently shown below.
+
+| Source            | Event                                                |
+|-------------------|------------------------------------------------------|
+| All jobs          | Job exit with runtime information                    |
+| All jobs          | Job scheduled for fast restart                       |
+| Custom job        | Execution started                                    |
+| Recompression job | Recompression job completed                          |
+| Reorder job       | Chunk reorder completed                              |
+| Reorder job       | Chunk reorder started                                |
+| Scheduler         | New jobs discovered and added to scheduled jobs list |
+| Scheduler         | Scheduling job for launch                            |
+
+### Debug level 2
+
+The amount of information printed at each level varies between jobs,
```

Comment:
It should be clear, but we could mention, that `DEBUG2` includes the messages of `DEBUG1`

Reply:
Added a sentence about that.